Updated AffectsVersion image (#149)
openjdk/guide: src/guide/jbs-jdk-bug-system.md

![Guidelines for setting [Affects Version/s]{.jbs-field}](affects_versions.svg)

> 1. Affects Version contains 8 only, which means this issue is applicable to 8 and all future releases (until it's fixed).

Reviewer: ```suggestion
1. [Affects Version/s]{.jbs-field} contains 8 only, which means this issue is applicable to 8 and all future releases (until it's fixed).
```
Shall we mark-up the JBS field?

Author: Right. I meant to do that but pushed in a hurry when I left the office... 😅

> 2. Affects Version contains 8, 8u40, 9, and 12, yet 8 already implies that the later releases are affected.

Reviewer: I used strike-through to indicate 8u40, 9, and 12 aren't need, which could also be confusing. Therefore, it makes to explicitly state that 8u40, 9, and 12 should not be added to the Affects Version field.

> 3. Adding the occasional LTS release value is ok.

Reviewer: We should expand the description of case (3) in a similar way:

```suggestion
3. [Affects Version/s]{.jbs-field} contains 11 and 21. Adding the occasional LTS release version (21 in this example) is fine; 17 is implicitly affected as well as versions after 21.
```

> 4. Affects Version contains 8 and the issue is fixed in 11. The 12-na label indicates that the issue is not applicable to 12 and subsequent versions.

Reviewer: ```suggestion
4. Affects Version contains 8 and the issue is fixed in 11. The [12-na]{.jbs-label} label indicates that the issue is not applicable to 12 and subsequent versions.
```

> 5. Affects Version contains 8, the issue is fixed in 21 and is backported to 17. The label 11-wnf indicates that the fix will not be backported to 11 or 8.

Reviewer: ```suggestion
5. Affects Version contains 8, the issue is fixed in 21 and is backported to 17. The [11-wnf]{.jbs-label} label indicates that the fix will not be backported to 11 or 8.
```
Mark up the label; I also moved the words for consistency with case 4.
